flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergCommitter.java

```diff
@@ -157,7 +158,7 @@ class TestIcebergCommitter extends TestBase {
   protected static List<Object> parameters() {
     List<Object> parameters = Lists.newArrayList();
     for (Boolean isStreamingMode : new Boolean[] {true, false}) {
-      for (int formatVersion : new int[] {1, 2}) {
+      for (int formatVersion : TestHelpers.ALL_VERSIONS) {
```

Comment:
Someone who is a flink expert maybe @stevenzwu or @pvary let me know if it was intentional that this test not check all versions or not

Reply:
Here it is not intentional.